PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -946,6 +946,118 @@ void WindowsDlg::doCount()
 	SetWindowText(_hSelf,msg.c_str());
 }
 
+void WindowsDlg::doSort()
+{
+	size_t count = (_pTab != NULL) ? _pTab->nbItem() : 0;
+	auto currrentTabIndex = _pTab->getCurrentTabIndex();
+	NMWINDLG nmdlg;
+	nmdlg.type = WDT_SORT;
+	nmdlg.hwndFrom = _hSelf;
+	nmdlg.curSel = currrentTabIndex;
+	nmdlg.code = WDN_NOTIFY;
+	nmdlg.nItems = static_cast<UINT>(count);
+	nmdlg.Items = new UINT[count];

```

Comment:
In order to avoid a potential memory leak, you can use a RAII approach. Example:
```
std::vector<UINT> items(count);
nmdlg.Items = items.data();
```